libavcodec/vvc/vvc_ps.c

```diff
@@ -199,7 +199,24 @@ static int gci_parse(GeneralConstraintsInfo *gci, GetBitContext *gb, void *log_c
         gci->no_ladf_constraint_flag = get_bits1(gb);
         gci->no_virtual_boundaries_constraint_flag = get_bits1(gb);
         num_reserved_bits = get_bits(gb, 8);
-        for (i = 0; i < num_reserved_bits; i++) {
+        if (num_reserved_bits > 5) {
+            gci->all_rap_pictures_constraint_flag = get_bits1(gb);
+            gci->no_extended_precision_processing_constraint_flag = get_bits1(gb);
+            gci->no_ts_residual_coding_rice_constraint_flag = get_bits1(gb);
+            gci->no_rrc_rice_extension_constraint_flag = get_bits1(gb);
+            gci->no_persistent_rice_adaptation_constraint_flag = get_bits1(gb);
+            gci->no_reverse_last_sig_coeff_constraint_flag = get_bits1(gb);
+            num_additional_bits_used = 6;
+        } else {
+            gci->all_rap_pictures_constraint_flag = 0;
```

Comment:
not needed, entire sps buffer zeroed by https://github.com/ffvvc/FFmpeg/blob/main/libavcodec/vvc/vvc_ps.c#L1168